### DISCUSSION TITLE
"Recently edited/created Resources" adding snippets?

### DISCUSSION THREAD

**Discussion by q2apro** [evolution-cms/evolution#1913] [Ideas]
It would be great to not only see the list of "Recently edited/created Resources" but also the **last edited snippets**.

Thanks for consideration!

**Comment by risingisland** [comment]
I am sure a plugin or module could be made for this, which might be better, as this is more of a user/admin preference.
Others may prefer to have the same, but for Chunks, TVs, etc.
IMO

**Comment by risingisland** [comment]
Maybe this could be adapted for use with snippets, etc.:
https://github.com/Nicola1971/DashboardListDoc-widget

**Comment by Dmi3yy** [comment]
I  think this info need for content manager + in 3.0 we recommended create all extra in files and use Git for teamwork.
